roles/postgres/tasks/scale_down_deployment.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- name: Check for presence of Deployment
+  k8s_info:
+    api_version: apps/v1
+    kind: Deployment
+    name: "{{ ansible_operator_meta.name }}-api"
+    namespace: "{{ ansible_operator_meta.namespace }}"
+  register: this_deployment
+
+# TODO: Determine if other deployments need to be scaled down too or not
```

Comment:
I think activation workers now also have access to the database but it depends on the EDA version used

https://github.com/ansible/eda-server/blob/main/src/aap_eda/settings/default.py#L516-L518

Reply:
@dsavineau thanks!  I added in the 2 worker deployments and the scheduler deployments, all of which connect to the db. 

```
    - "{{ ansible_operator_meta.name }}-scheduler"
    - "{{ ansible_operator_meta.name }}-default-worker"
    - "{{ ansible_operator_meta.name }}-activation-worker"
 ```